### ISSUE TITLE
Erroneous URI's in output

### ISSUE THREAD

**Issue by kiska3** [ArchiveTeam/tumblr-grab#22]
Crawls are coming up with URI's such as:
http://yorkpud.tumblr.com/%F2%0CZ%AB%C84%C2%B2b%E8@%FE%BA%AF%A8~k%C8/Http://cumsblr.tumblr.com

These urls seem to be coming from malformed URI's even though they 404

So http://yorkpud.tumblr.com/%B1%BEM/Http://cumsblr.tumblr.com was discovered from http://yorkpud.tumblr.com/%B1%BEM/z%F0Hj%D5%FE+%EB%E5%F6_%9F%C4%CC

**Comment by kiska3** [comment]
Should be fixed with this commit: https://github.com/ArchiveTeam/tumblr-grab/commit/f2c70e0db80923d6afb1d1fc70fde3e6caa41134

**Comment by marked** [comment]
Isn't this the one I found a regression on?

**Comment by kiska3** [comment]
> Isn't this the one I found a regression on?

Yes however I am not too sure which commit successfully excludes those urls
